From a6d76881a06350e807107bdd71d62efee4df485e Mon Sep 17 00:00:00 2001
Message-ID: <a6d76881a06350e807107bdd71d62efee4df485e.1760527144.git.alx@kernel.org>
In-Reply-To: <d6d3123c7271c11fc403906ee3971b22c2fe8e4c.1760476615.git.alx@kernel.org>
References: <d6d3123c7271c11fc403906ee3971b22c2fe8e4c.1760476615.git.alx@kernel.org>
From: Alejandro Colomar <alx@kernel.org>
Date: Tue, 14 Oct 2025 23:15:28 +0200
Subject: [PATCH v3] CONTRIBUTING.d/ai: Add guidelines banning AI for
 contributing
To: <linux-man@vger.kernel.org>
Cc: Carlos O'Donell <carlos@redhat.com>,
    Collin Funk <collin.funk1@gmail.com>

This policy is based on the Gentoo policy (see link below).

Cc: Carlos O'Donell <carlos@redhat.com>
Cc: Collin Funk <collin.funk1@gmail.com>
Cc: Sam James <sam@gentoo.org>
Link: <https://wiki.gentoo.org/wiki/Project:Council/AI_policy>
Signed-off-by: Alejandro Colomar <alx@kernel.org>
---

Hi!

I changed obvious wording, such as replacing the project name.
On top of that, I removed a misleading sentence:

	They are capable of providing good assistance if you are careful
	enough, but we can't really rely on that.

I don't think it's possible to be careful enough.


Have a lovely day!
Alex

 CONTRIBUTING.d/ai | 42 ++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 42 insertions(+)
 create mode 100644 CONTRIBUTING.d/ai

diff --git a/CONTRIBUTING.d/ai b/CONTRIBUTING.d/ai
new file mode 100644
index 000000000..92d5d85d6
--- /dev/null
+++ b/CONTRIBUTING.d/ai
@@ -0,0 +1,42 @@
+Name
+	AI - using artificial intelligence for contributing
+
+Description
+	It is expressly forbidden to contribute to this project any
+	content that has been created with the assistance of artificial
+	intelligence tools.
+
+    Concerns
+	Copyright concerns.
+		At this point, the regulations concerning copyright of
+		generated contents are still emerging worldwide.  Using
+		such material could pose a danger of copyright
+		violations, but it could also weaken claims to copyright
+		and void the guarantees given by copyleft licensing.
+
+	Quality concerns.
+		Popular LLMs are really great at generating plausibly
+		looking, but meaningless content.  At this point, they
+		pose both the risk of lowering the quality of a project,
+		and of requiring an unfair human effort from
+		contributors and maintainers to review contributions and
+		detect the mistakes resulting from the use of AI.
+
+	Ethical concerns.
+		The business side of AI boom is creating serious ethical
+		concerns.  Among them:
+
+		-  Commercial AI projects are frequently indulging in
+		   blatant copyright violations to train their models.
+		-  Their operations are causing concerns about the huge
+		   use of energy and water.
+		-  The advertising and use of AI models has caused
+		   a significant harm to employees and reduction of
+		   service quality.
+		-  LLMs have been empowering all kinds of spam and scam
+		   efforts.
+
+Caveats
+	This policy can be revisited, should a case been made over such
+	a tool that does not pose copyright, ethical, and quality
+	concerns.

base-commit: ac6f5c32b3fae7549c5a42d96a3273adc24e5023
-- 
2.51.0

